IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowseTests.LeftPanel.Remotes.cs

```diff
@@ -59,8 +59,11 @@ public void SetUp()
         }
 
         [TearDown]
-        public void TearDown()
+        public async Task TearDown()
```

Comment:
Should we call this 
```suggestion
        public async Task TearDownAsync()
```

Reply:
I'll test to verify that the name doesn't impact whether the method gets called.

Reply:
Fixed these!